### DISCUSSION TITLE
Bank Setting to set MIDI Clock Tempo per bank

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#45] [Feature Requests & Ideas]
Originally submitted on Looped.in ideas page with 42 community votes.

**Comment by runeflobakk** [comment]
Could this be implemented as a Smart Message, e.g. "Set Tempo"? Then one would have the flexibility to change to pre-set tempos wherever it is possible to assign messages. To have pre-set tempos stored per-bank, just assign a Set Tempo-message as a Bank Message.
